metrics/sink.go

```diff
@@ -85,9 +80,17 @@ type TrendSink struct {
 // IsEmpty indicates whether the TrendSink is empty.
 func (t *TrendSink) IsEmpty() bool { return t.Count == 0 }
 
+// Add adds a sample to the TrendSink.
+//
+// Note that this method does not compute the median value. The
+// operation is delegated to the Calc() method, which is called
+// automatically when the sink is formatted, or when thresholds
+// are checked. The reason is that calculating the median value
+// is an expensive operation, and we don't want to do it for
+// every sample.
```

Comment:
```suggestion
```

This is no longer true and with the removing of the `Med` field will also be unobservable